code/datums/traits/good.dm

```diff
@@ -183,4 +183,68 @@
 	mob_trait = TRAIT_VORACIOUS
 	gain_text = "<span class='notice'>You feel HONGRY.</span>"
 	lose_text = "<span class='danger'>You no longer feel HONGRY.</span>"
-	medical_record_text = "Patient demonstrates a disturbing capacity for eating."
\ No newline at end of file
+	medical_record_text = "Patient demonstrates a disturbing capacity for eating."
+
+/datum/quirk/accent
+	name = "Accent"
+	desc = "You have developed an accent."
+	value = -1
+	mob_trait = TRAIT_RANDOM_ACCENT
```

Comment:
mob_trait = TRAIT_ACCENT

Reply:
Sorry, I'm keeping this for debug testing. 
It's a WIP. 
Why are you even reviewing this right now its clearly a WIP? Learn to read.

Reply:
Because I like you
Also this is just plain broken and needs to be fixed, not even debug testing